docs/reference/commandline/events.md

```diff
@@ -51,6 +51,7 @@ The currently supported filters are:
 * container
 * event
 * image
+* labels
```

Comment:
but we don't currently have a filter, that uses `--filter=<key>=<name>=<value>`, at least, no example, because with labels the user is able to set both the *name* and the *value* of the label.